### ISSUE TITLE
Disable email checking by default?

### ISSUE THREAD

**Issue by mre** [lycheeverse/lychee#575]
Most link checkers don't support checking email addresses.
On top of that, it can be hard to decide what a proper email address looks like. E.g. `php@7.2` would be a false-positive, which could be a valid email address.

We should consider disabling email checks by default, deprecating the `--exclude-mail` flag and adding an `--include-mail` flag instead. Alternatively we disable it in the Github action.

**Comment by lebensterben** [comment]
https://en.m.wikipedia.org/wiki/Jamie_Zawinski#Zawinski's_Law

> Every program attempts to expand until it can read mail.

this feature sounds scary when I first saw it.

**Comment by polarathene** [comment]
I was surprised to see email addresses being checked by default (_but I have noticed the opt-out option for this type documented_). I would assume it's rare that users want those checked (especially when not `mailto:` values), and opt-in would make more sense?

I plan to use lychee with docs for a mail-server product, and one of the inputs had quote wrapped example addresses. `lychee` was capturing these as `'user@example.com` (_including the left quote, but ignoring the right-side one_).

**Comment by mre** [comment]
Yeah, excluding it by default is probably the way to go. At least I lean towards that at the moment.

**Comment by thomas-zahner** [comment]
@mre This seems to be the default behaviour since 04887ee2936485943e9c55fef9122e9927e00e74
I think we should either close this issue or reuse (and rename) it for removing the now deprecated option `--exclude-mail`.

**Comment by mre** [comment]
Yes, let's close it. Can you create a separate issue for the deprecation? Easier to keep track of it this way.
